airflow/security/utils.py

```diff
@@ -56,26 +55,3 @@ def replace_hostname_pattern(components, host=None):
     if not fqdn or fqdn == '0.0.0.0':
         fqdn = get_hostname()
     return '%s/%s@%s' % (components[0], fqdn.lower(), components[2])
-
-
-def get_fqdn(hostname_or_ip=None):
```

Comment:
Shoudln't we deprecate them first? I tthink someone could have used those method - they seem generic enough to be usable.

Reply:
Agree with @potiuk . It makes more sense to deprecate first.